security/tpm/test/_tpm_test.js

```diff
@@ -97,11 +100,30 @@ describe('tpm', function () {
       testFalsyArg('signWithIdentity', 'dataToSign', falsyDataToSign, ReferenceError);
     });
 
-    it('must call activateIdentityKey first', function() {
-      /*Tests_SRS_NODE_TPM_SECURITY_CLIENT_06_013: [If `signWithIdentity` is invoked without a previous successful invocation of `activateIdentityKey`, an InvalidOperationError is thrown.] */
+    it('must call activateIdentityKey first', function(done) {
+      /*Tests_SRS_NODE_TPM_SECURITY_CLIENT_06_013: [If `signWithIdentity` is invoked without a previous successful invocation of `activateIdentityKey`, the callback will be invoked with `err` of `InvalidOperationError`.] */
```

Comment:
so I guess "previous successful invocation" now applies to the lifecycle of the TPM but not the TPM security client state machine itself, right?

Reply:
Yes, somewhere in the depths of time, activateIdentityKey needs to be called.